### PR TITLE
Support for ingress resource in meshsync

### LINKED ISSUE #217: Support Ingresses
**Current Behavior**
MeshSync doesn't discover this resource type.

**Implementation**
Add support for discovering this Kubernetes resource type. Do the same thing that was done here - https://github.com/meshery/meshsync/commit/1b640b8cbfd6ffefc0896de8fc1ac2a6bd284822, but for `ingresses`

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). 
- Fill-in a [Community Member Form](https://layer5.io/newcomer) and join the [Community Slack](http://slack.meshery.io) for access.

### PR DESCRIPTION
**Pull request by Vadiraj-Puranik**
Signed-off-by : vadiraj.puranik08@gmail.com 
**Description**

This PR fixes #217

**Notes for Reviewers**
1:Updated the default_config.go to support ingress resource

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: internal/config/default_config.go [hunk 1]
```diff
@@ -66,6 +66,11 @@ var (
 				Name:      "daemonsets.v1.apps",
 				PublishTo: "meshery.meshsync.core",
 			},
+			//Added Ingress support
+			{
+   				Name:      "ingresses.v1.networking.k8s.io",
```
Comment (leecalcote) [inline internal/config/default_config.go:71]:
```suggestion
   				Name:      "ingresses.v1.",
```

It'd be good to capture Ingresses a little more broadly.

**Comment by leecalcote** [comment]
Excellent, @Vadiraj-Puranik 👍

**Comment by leecalcote** [comment]
@Vadiraj-Puranik please run `gofmt -w -s -l .` on future commits.

**Comment by Vadiraj-Puranik** [comment]
> @Vadiraj-Puranik please run `gofmt -w -s -l .` on future commits.

@leecalcote  sure will keep this in mind

**Pull request merged**